analog-in/maven/src/main/java/iotdk/example/AnalogIn.java

```diff
@@ -38,10 +38,8 @@
 
 	// Set true if using a Grove Pi Shield, else false
     static final boolean USING_GROVE_PI_SHIELD = true;
-	static String unknownPlatformMessage = "This sample uses the MRAA/UPM library for I/O access, " +
-		"you are running it on an unrecognized platform. " +
-		"You may need to modify the MRAA/UPM initialization code to " +
-		"ensure it works properly on your platform.\n\n";
+	
+	static int pinNumber = 13;
```

Comment:
13 is a poor default choice for analog pins. Typically you'll have 0 through 5 available as analog on Arduino-like boards.